Sources/BraveWallet/Crypto/Transaction Confirmations/StaticTextView.swift

```diff
@@ -0,0 +1,44 @@
+// Copyright 2023 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+
```

Comment:
Extra whitespace

Reply:
update in [02e39dd](https://github.com/brave/brave-ios/pull/6960/commits/02e39dd3500416bf27c899accce6252b59e78d3f)